internal/syncer/git_commit_stats.go

```diff
@@ -190,7 +190,7 @@ func (w *worker) handleGitCommitStats(ctx context.Context, j *db.DequeueSyncJobR
 			}, nil
 		}, libgit2.DiffDetailLines)
 		if err != nil {
-			w.logger.Err(err).Msgf("error iterating over diff")
+			w.logger.Warn().AnErr("Error", err).Msgf("error iterating over diff")
```

Comment:
I think this is another one we'll want to know about/alert on, since it's somewhat "application level"